jenkins-slaves/jenkins-slave-conftest/README.md

```diff
@@ -0,0 +1,16 @@
+# Jenkins Reqo Slave 🦇
```

Comment:
Reqo? typo?

Reply:
lolz - I initially called it a rego slave but ovs find and replace didn't catch this one